OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -336,11 +330,10 @@ public void Activate(Player p)
 			Player = p;
 			IsEnabled = true;
 			playerPower = p.PlayerActor.Trait<PowerManager>();
-			supportPowerMngr = p.PlayerActor.Trait<SupportPowerManager>();
 			playerResource = p.PlayerActor.Trait<PlayerResources>();
-			frozenLayer = p.PlayerActor.Trait<FrozenActorLayer>();
 
 			harvManager = new AIHarvesterManager(this, p);
```

Comment:
Would you mind renaming `aiHarvManager` too for consistency?  Or I guess we could go back to `supportPowerManager` as the ambiguity is now solved.